app/javascript/mastodon/utils/icons.jsx

```diff
@@ -1,7 +1,3 @@
-import React from 'react';
-
-// Copied from emoji-mart for consistency with emoji picker and since
-// they don't export the icons in the package
```

Comment:
I think these comments are still relevant.

Reply:
Good catch, I am not sure why it has been deleted